source/resampling_with_code2.Rmd

```diff
@@ -25,9 +25,9 @@ source("_common.R")
 
 # More resampling with code
 
-@sec-resampling code introduced a problem in probability, that was also a problem in statistics.  We asked how surprised we should be the results of a trial of a new cancer treatment regime.
+@sec-resampling-code introduced a problem in probability, that was also a problem in statistics.  We asked how surprised we should be at the results of a trial of a new cancer treatment regime.
 
-Here we study another urgent problem in the real world - racial bias and the
+Here we study another urgent problem in the real world—racial bias and the
```

Comment:
I think the original was right ...  Do you mean "real-world racial bias"?  

```suggestion
Here we study another urgent problem in the real world - racial bias and the
```

Reply:
Ah, yes, I am used to https://www.thepunctuationguide.com/em-dash.html#:~:text=The%20em%20dash%20is%20typically,and%20after%20the%20em%20dash.